helm/promtail/templates/service.yaml

```diff
@@ -19,7 +19,6 @@ spec:
     {{- include "promtail.selectorLabels" $ | nindent 4 }}
   clusterIP: "None"
   ports:
-    - protocol: TCP
-      port: 3101
-      targetPort: http-metrics
+    - name: http
```

Comment:
Do we still need this if we use the upstream service monitor?

Reply:
No you're right.